src/features/todos.ts

```diff
@@ -1,8 +1,55 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
+import { getTodos } from '../api';
+import { RootState } from '../app/store';
+
+export interface Todos {
+  todos: Todo[];
+  loaded: boolean;
+  hasError: boolean;
+}
+
+const initialState: Todos = {
+  todos: [],
+  loaded: true,
+  hasError: false,
+};
+
+export const loadTodoFromServer = createAsyncThunk<Todo[]>(
+  'todos/getTodosFromServer',
+  () => getTodos(),
+);
```

Comment:
The task description specifies that todos should be loaded in the `App` on page load without using Redux Thunk. However, the `loadTodoFromServer` function is implemented using `createAsyncThunk`, which is part of Redux Thunk. Consider revising this approach to align with the task requirements.